integration_tests/test_fine_tunes.py

```diff
@@ -21,11 +24,20 @@ def test_fine_tunes() -> None:
     # num_jobs = len(list_response_0_before["jobs"])
     # assert num_jobs >= 1
 
-    list_response_1 = list_fine_tunes(USER_ID_1)
-    assert len(list_response_1["jobs"]) == 0
-
-    # cancel_response = cancel_fine_tune_by_id(fine_tune_id, USER_ID_0)
-    # assert cancel_response["success"]
+    # list_response_1 = list_fine_tunes(USER_ID_0)
```

Comment:
there should be some `@pytest.skip(reason="test doesn't work")` thing that you can do to skip the test (as opposed to commenting it out)

Reply:
yeah let me add this too - although I will say the test was already doing nothing